tests/programs/clinical-submission.js

```diff
@@ -134,29 +142,23 @@ module.exports = {
     browser.expect.element('#button-clear-submission').to.have.attribute('disabled');
   },
 
-  'Submission - Upload Bad Clinical Data, Clear Selected File Upload, Validate Error, Clear Selected File, Validate': browser => {
+  'Submission - Upload Bad Clinical Data, Validate Error, Clear Selected File, Validate': browser => {
     startAsUser(browser)(TEST_USERS.DCC_ADMIN)
       .perform(async () => {
         await submitClinicalData({
           jwt: TEST_USERS.DCC_ADMIN.token,
           shortName: program.shortName,
-          good: 'false',
+          good: false,
         });
       })
```

Comment:
asynchronous operations are handled with a `done` callback by nightwatch https://nightwatchjs.org/api/perform.html#apimethod-page let's update to match this API

Reply:
also try removing all the `.pause` after these asynchronous stuff once confirmed that works